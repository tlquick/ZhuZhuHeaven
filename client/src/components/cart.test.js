import { render, screen, cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";
import Cart from "./cart";
afterEach(() => {
  cleanup();
});
test("renders cart page", () => {
  render(<Cart />);
  const cartElement = screen.getByTestId("cart-1");
  expect(cartElement).toBeInTheDocument();
  expect(cartElement).toContainHTML("div"); //contains a <div> tag
});
test("matches cart snapshot", () => {
  const tree = renderer.create(<Cart />).toJSON();
  expect(tree).toMatchSnapshot();
});
