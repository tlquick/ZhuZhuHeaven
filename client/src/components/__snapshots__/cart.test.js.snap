// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches cart snapshot 1`] = `
<div
  className="main-screen"
  data-testid="cart-1"
>
  Cart - check signed in
</div>
`;
